great-idea/services.html

```diff
@@ -15,93 +15,105 @@
 </head>
 
 <body>
+    <div class="container">
+        <header>
+         <nav>
+          <a href="#">Services</a>
+          <a href="#">Product</a>
+          <a href="#">vision</a>
+          <a href="#">Features</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+         </nav>
 
+         <img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
+</header>
 
-Services
-Product
-Vision
-Features
-About
-Contact
```

Comment:
I would everything from about here to the flex bottom in a `main` tag and then put padding maybe 100px or so will stop that header nav from overlapping it